doc/extending/optimization.txt

```diff
@@ -221,8 +223,9 @@ computation, using the ``merge_optimizer`` defined in
 
 >>> from theano.gof.opt import merge_optimizer
 >>> merge_optimizer.optimize(e)
+(0, 0.0001430511474609375, None, None, {}, 1, 0)
```

Comment:
This include a timing (the second element of the tuple) that should be replaced with `...` and `#  +doctest: ELLIPSIS`